maps-db/src/main/resources/db/migration/V1.00.01__create_map_index_table.sql

```diff
@@ -0,0 +1,33 @@
+comment on database maps_db is 'Database used to store, track and serve map information';
+
+create table map (
+  id serial primary key,
+  name varchar(256) not null unique check(length(name) > 3),
+  date_created timestamptz not null default now(),
```

Comment:
Not sure we need a creation date, but I guess it doesn't hurt

Reply:
Generally all rows should have both date_created and date_updated. Missing date_updated implies immutability. TBD how we handle deleted or removed maps. The value here is if a version were to go away when we could see when the map was originally uploaded.